guidelines.md

```diff
@@ -81,96 +81,47 @@ Create a test for your component(s) located in the [/src/__tests__](/src/__tests
 
 ### Internationalization (i18n)
 
-All text displayed to the user should be internationalized. We currently use [react-intl](https://github.com/formatjs/react-intl) for internationalization of our app. There are [helper functions](https://github.com/cyverse-de/ui-lib/blob/master/src/util/I18NWrapper.js) in the [ui-lib](https://github.com/cyverse-de/ui-lib) repo that should be used.
+All text displayed to the user should be internationalized. We currently use [i18next](https://www.i18next.com) and [next-i18next](https://github.com/isaachinman/next-i18next) for internationalization of our app.
 
 Generally, you'll need to:
 
-1. Create a `messages` file similar to the following with all the text your component(s) will need. Generally, the first 3 lines don't change - `locales` and `messages` must be present. Beyond that, you have your text in key-value pairs.
+1. Create a `translation` json file under `/public/static/locales/en` similar to the following with all the text your component(s) will need. The files needs be in JSON format and be saved with `.json` extension.
 
 ```
-export default {
-    locales: "en-US",
-    messages: {
-        header: "Welcome",
-        namedHeader: "Welcome back {name}",
-        stringTitle: "Something that's not an object",
+{
+    "cyverse": "CyVerse",
+    "dashboard": "Dashboard",
+    "welcome" : "Welcome {{name}}",
+    "data": "Data",
+    "deTitle": "Discovery Environment",
+    "discovery": "<b>D</b>iscovery"
     }
-}
-```
-
-2. Export your component while wrapping it with the `withI18N` [higher-order component](https://reactjs.org/docs/higher-order-components.html) from the [helper function](https://github.com/cyverse-de/ui-lib/blob/master/src/util/I18NWrapper.js) file in the `ui-lib` repo and pass in the `messages` file you created. This basically links them together so `react-intl` knows where to look up text.
-
-```
-import React from "react";
-+import intlData from "./messages";
-+import { withI18N } from "@cyverse-de/ui-lib";
-
-function MyComponent(props) {
-    return (
-        <>
-            <span>Welcome</span>
-            <span>Welcome back {props.name}</span>
-        </>
-    )
-}
-
--export default MyComponent;
-+export default withI18N(MyComponent, intlData);
-
 ```
 
-3. Update your component to fetch the internationalized text with the corresponding key you created. The [getMessage(key)](https://github.com/cyverse-de/ui-lib/blob/78880901c263c14ea697a5abd9b607fbd776ec4b/src/util/I18NWrapper.js#L29-L34) helper function covers roughly 80% of the use cases
+2. Import all the translation json file(s) into your react component with filename used as namespace. For example, if you have named the translation files `common.json` and `search.json`, you can import those translations as 
```

Comment:
There should also be mention of what needs to get added on the page level with `getInitialProps` I believe?

Reply:
That right. I was in double mind on whether to put it here and make the instructions complicated. I will go ahead and add it.